packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -2398,7 +2398,7 @@ describe('SnapController', () => {
           },
         }),
       ).rejects.toThrow(
-        'Assertion failed: At path: foo -- Expected a value of type `never`, but received: `"bar"`.',
+        'Assertion failed: Expected the value to satisfy a union of `object | object`, but received: [object Object].',
```

Comment:
Hmm, I wonder why this error message changed to be worse. Do you know?

Reply:
It does not change anything